src/freenet/support/SortedIntSet.java

```diff
@@ -221,12 +191,11 @@ public synchronized void clear() {
 	 * @return sorted array of all items
 	 */
 	public synchronized int[] toArrayRaw() {
```

Comment:
It is used in the Spider and Library plugin. If I were to remove it, I could as well have removed the entire class immediately ;)